lib/csv_to_popolo.rb

```diff
@@ -229,135 +231,114 @@ def legislatures
       legislative_memberships.count.zero? ? [] : [
         {
           id: 'legislature',
-          name: 'Legislature', 
+          name: 'Legislature',
           classification: 'legislature',
-          legislative_periods: terms,
-        }.select { |_, v| !v.nil? } 
+          legislative_periods: terms
+        }.select { |_, v| !v.nil? }
       ]
     end
 
     def executive
       executive_memberships.count.zero? ? [] : [{
         id: 'executive',
-        name: 'Executive', 
-        classification: 'executive',
+        name: 'Executive',
+        classification: 'executive'
       }]
     end
 
-    def legislative_memberships 
-      @_lmems ||= @csv.find_all { |r| r.has_key? :group }.map do |r|
-        mem = { 
+    def legislative_memberships
```

Comment:
Assignment Branch Condition size for legislative_memberships is too high. [24.7/15]